docs/java_cli_for_managedgateway.rst

```diff
@@ -145,8 +145,8 @@ The gateway can invoke sendDeviceManageRequest() method to make the attached dev
 
 .. code:: java
 
-	managedGateway.sendGatewayManageRequest(typeId, deviceId, lifetime, true, true);
-	
+	sendDeviceManageRequest(typeId, deviceId, lifetime, true, true);
```

Comment:
@sathipal 
The lead-in statement implies that the sample should contain sendDeviceManageRequest instead?
is this code sample correct here?
Thanks